_src/src/i18n.js

```diff
@@ -0,0 +1,47 @@
+const supportLanguage = [ "zh-cn", "en" ];
```

Comment:
Can we somehow get rid of this hardcoded array and just try to load language during runtime?

Reply:
Now I was change the loader to dynamic load language file, that can load language if file exists